Makefile

```diff
@@ -1,4 +1,5 @@
 include ./commons-test.mk
+include mk/dependabot.mk
```

Comment:
Could you move the directory to be hidden: `.mk`?

BTW I liked this approach and we should move all Make helper files into that folder.

Reply:
Done !